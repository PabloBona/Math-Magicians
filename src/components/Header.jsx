import React from 'react';
import '../App.css';

const Header = () => (
  <nav className="navbar navbar-expand-lg navbar-collapse-down header bg-black">
    <div className="container bg-black py-2">
      <h1 className="text-center text-light">
        Math
        {' '}
        <span className="carrot">Magicians</span>
      </h1>
      <button className="navbar-toggler bg-body-secondary" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
        <span className="navbar-toggler-icon bg-body-secondary" />
      </button>
      <div className="collapse navbar-collapse justify-content-end bg-black" id="navbarNavAltMarkup">
        <div className="navbar-nav bg-black">
          <a className="nav-link text-light separate fs-5 header-carrot" aria-current="page" href="/">Home</a>
          <a className="nav-link text-light separate fs-5 header-carrot" href="/calc">Calculator</a>
          <a className="nav-link text-light separate fs-5 header-carrot" href="/quotes">Quotes</a>
        </div>
      </div>
    </div>
  </nav>
);

export default Header;
