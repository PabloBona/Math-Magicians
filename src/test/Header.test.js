import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

describe('Header component unit test', () => {
  test('Header Component Renders Properly', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
