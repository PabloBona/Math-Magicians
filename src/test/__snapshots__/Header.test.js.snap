// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component unit test Header Component Renders Properly 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-collapse-down header bg-black"
>
  <div
    className="container bg-black py-2"
  >
    <h1
      className="text-center text-light"
    >
      Math
       
      <span
        className="carrot"
      >
        Magicians
      </span>
    </h1>
    <button
      aria-controls="navbarNavAltMarkup"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler bg-body-secondary"
      data-bs-target="#navbarNavAltMarkup"
      data-bs-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon bg-body-secondary"
      />
    </button>
    <div
      className="collapse navbar-collapse justify-content-end bg-black"
      id="navbarNavAltMarkup"
    >
      <div
        className="navbar-nav bg-black"
      >
        <a
          aria-current="page"
          className="nav-link text-light separate fs-5 header-carrot"
          href="/"
        >
          Home
        </a>
        <a
          className="nav-link text-light separate fs-5 header-carrot"
          href="/calc"
        >
          Calculator
        </a>
        <a
          className="nav-link text-light separate fs-5 header-carrot"
          href="/quotes"
        >
          Quotes
        </a>
      </div>
    </div>
  </div>
</nav>
`;
